gbpservice/nfp/configurator/agents/loadbalancer_v1.py

```diff
@@ -80,10 +81,28 @@ def update_pool_stats(self, pool_id, stats, context):
                'method': 'update_pool_stats',
                'kwargs': {'context': context.to_dict(),
                           'pool_id': pool_id,
-                          'stats': stats}
+                          'stats': stats,
+                          'pool': pool_id}
```

Comment:
pool_id is already available in the dict. Why is this passed with different key again?